jenkins/deployment/k8s_deployment/cortx-cluster.groovy

```diff
@@ -21,7 +21,7 @@ pipeline {
         string(name: 'CORTX_ALL_IMAGE', defaultValue: 'ghcr.io/seagate/cortx-all:2.0.0-latest', description: 'CORTX-ALL image', trim: true)
```

Comment:
Please remove CORTX-ALL image from the CORTX Deployment Jenkins pipelines.

Reply:
Done